core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -24,7 +24,7 @@ <h4 style="clear: left; display: inline-block;">Summary</h4>
         <th></th>
         <th>RDD Blocks</th>
         <th><span data-toggle="tooltip"
-                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory</span>
+                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory used/total</span>
```

Comment:
The memory data is separated from each other, and if the title is separated, it is easy to observe and understand the user.Modify this slight UI, advantages outweigh the disadvantages.